src/main/java/org/jitsi/videobridge/datachannel/RemotelyOpenedDataChannel.java

```diff
@@ -32,10 +33,11 @@
      */
     public RemotelyOpenedDataChannel(
             DataChannelStack.DataChannelDataSender dataChannelDataSender,
+            Logger parentLogger,
```

Comment:
Usually parentLogger is added as the last argument, but here you have it as the second. Any reason?

Reply:
i haven't necessarily sought to always set it as the last arg (in kotlin especially, where it's best  to put args with default values last) but i can't necessarily  explain why i put it second here. i  don't feel strongly about it.

Reply:
Ok, as you like. As things move to Kotlin the arg order matters less anyway.
LGTM.